shared/app/index.native.tsx

```diff
@@ -9,8 +9,12 @@ import {makeEngine} from '../engine'
 import {GestureHandlerRootView} from 'react-native-gesture-handler'
 import {enableFreeze} from 'react-native-screens'
 import {setKeyboardUp} from '../styles/keyboard-state'
+import {setServiceDecoration} from '../common-adapters/markdown/react'
+import ServiceDecoration from '../common-adapters/markdown/service-decoration'
 enableFreeze(true)
 
+setServiceDecoration(ServiceDecoration)
```

Comment:
service decoration has a lot of non common specific components so to avoid circular deps lets just pass this in later